Source/WebCore/page/EventHandler.h

```diff
@@ -249,7 +249,7 @@ class EventHandler : public CanMakeCheckedPtr {
     bool dispatchGestureEvent(const PlatformTouchEvent&, const AtomString&, const EventTargetSet&, float, float);
 #elif ENABLE(MAC_GESTURE_EVENTS)
     bool dispatchGestureEvent(const PlatformGestureEvent&, const AtomString&, const EventTargetSet&, float, float);
-    WEBCORE_EXPORT bool handleGestureEvent(const PlatformGestureEvent&);
+    WEBCORE_EXPORT HandleUserInputEventResult handleGestureEvent(const PlatformGestureEvent&);
```

Comment:
We’ll need to do this eventually (so maybe it’s fine), but there’s no reason for this to change in this PR. The plumbing isn’t in place for `RemoteUserInputEventData` to be sent to the UI process for gesture events, so we only need the bool.

Reply:
Yes I agree! I just made the change to resemble the other event handling methods, but I agree we don't need this interface to change. I'll remove this change for now.

Reply:
Addressed in https://github.com/WebKit/WebKit/commit/24ca43ebc6b8bc9e932394053c5e6640c740509d